ada/testsuite/ada/nameres.adb

```diff
@@ -156,6 +160,12 @@ procedure Nameres is
          Help => "Show the time it took to nameres each file. "
                  & "Implies --quiet so that you only have timing info");
 
+      package Memory is new Parse_Flag
+        (App.Args.Parser,
+         Long => "--memory",
+         Short => "-T",
+         Help => "Show the global memory footprint on exit. ");
```

Comment:
Please remove the extra space after the dot and align the arrows.

Reply:
Addressed this and all your other comments on the first patch.

Reply:
Great, thanks a lot